### DISCUSSION TITLE
help - using libvips in wolfi os Docker image

### DISCUSSION THREAD

**Discussion by jsulbaran** [libvips/libvips#4295] [General]
Hi! in my company we're trying to  use libvips (with govips) in a docker image based on [wolfi os](https://github.com/wolfi-dev/os). We can manage to install the lib but we only get jpeg and gif basic support, every other format is not listed as supported even when libs and .pc files are found.

Could you please give some advice to progress ? 

Thank you in advance!!

some data:

#pkg-config vips --cflags --libs :
-I/usr/include/libmount -I/usr/include/blkid -pthread -I/usr/include/glib-2.0 -I/usr/lib/glib-2.0/include -lvips -lgio-2.0 -lgobject-2.0 -lglib-2.0

bash-5.2# vips --vips-config
enable debug: false
enable deprecated: true
enable modules: true
enable cplusplus: true
enable RAD load/save: true
enable Analyze7 load: true
enable PPM load/save: true
enable GIF load: true
FFTs with fftw: false
SIMD support with libhwy or liborc: false
ICC profile support with lcms2: false
deflate compression with zlib: true
text rendering with pangocairo: false
font file support with fontconfig: false
EXIF metadata support with libexif: true
JPEG load/save with libjpeg: true
JXL load/save with libjxl: false (dynamic module: false)
JPEG2000 load/save with OpenJPEG: false
PNG load/save with libspng or libpng: false
image quantisation with imagequant or quantizr: false
TIFF load/save with libtiff: false
image pyramid save with libarchive: false
HEIC/AVIF load/save with libheif: false (dynamic module: false)
WebP load/save with libwebp: false
PDF load with PDFium or Poppler: false (dynamic module: false)
SVG load with librsvg: false
EXR load with OpenEXR: false
WSI load with OpenSlide: false (dynamic module: false)
Matlab load with Matio: false
NIfTI load/save with libnifti: false
FITS load/save with cfitsio: false
GIF save with cgif: false
Magick load/save with MagickCore: false (dynamic module: false)

installation steps:

RUN apk add --no-scripts tiff-dev tiff libjpeg-turbo-dev libarchive-dev pkgconf libheif-dev libpng-dev libpng-static libpng
RUN export PKG_CONFIG_PATH=/usr/lib/pkgconfig:$PKG_CONFIG_PATH
RUN export G_MESSAGES_DEBUG=VIPS
RUN ldconfig
RUN apk add --no-scripts libvips libvips-dev
RUN  pkg-config vips --cflags --libs

**Comment by lovell** [comment]
This is a question for the maintainers of the Wolfi-provided libvips package as defined by https://github.com/wolfi-dev/os/blob/main/libvips.yaml

Please see https://github.com/libvips/libvips#optional-dependencies for a list of possible, additional, optional dependencies.
